config/tikv/compaction.toml

```diff
@@ -0,0 +1,23 @@
+# maintainer: cosven
+
+[gc]
+enable-compaction-filter = true
+compaction-filter-skip-version-check = true
+
+[coprocessor]
+consistency-check-method = "mvcc"
+
+[raftstore]
+consistency-check-interval = "180s"
+
+[rocksdb]
+force-consistency-checks = true
+
+[rocksdb.defaultcf]
+enable-compaction-guard = true
```

Comment:
let's also set `target-file-size-base = 128M`. This config will default to 128M when we default on compaction guard, but not right now.